pkg/build/addimage/defaults.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Kubernetes Authors.
+Copyright 2024 The Kubernetes Authors.
```

Comment:
Not sure why git sees this as a moved file. It is net-new, so diff is incorrect about changing copyright dates.